internal/api/v1beta1/alert.go

```diff
@@ -42,11 +42,33 @@ func (s *GRPCServer) ListAlerts(ctx context.Context, req *sirenv1beta1.ListAlert
 }
 
 func (s *GRPCServer) CreateAlerts(ctx context.Context, req *sirenv1beta1.CreateAlertsRequest) (*sirenv1beta1.CreateAlertsResponse, error) {
-	createdAlerts, firingLen, err := s.alertService.CreateAlerts(ctx, req.GetProviderType(), req.GetProviderId(), req.GetBody().AsMap())
+	items, err := s.createAlerts(ctx, req.GetProviderType(), req.GetProviderId(), 0, req.GetBody().AsMap())
```

Comment:
why the namespace id is a hardcoded `0`?

Reply:
this api won't be used anymore and just added as a fallback option, the 0 namespace id should be translated to null in the db